index.html

```diff
@@ -54,20 +54,20 @@
 
 <p>The Gregorio project offers tools for typesetting Gregorian chant. These tools include:</p>
 <ul>
-<li><a href="/gabc/">gabc</a>: a notation for representing Gregorian chant using ASCII characters</li>
-<li><a href="/gregoriotex/">GregorioTeX</a>: a TeX style for typesetting scores</li>
-<li>a software application to convert from gabc to GregorioTeX</li>
+<li><a href="./gabc/">gabc</a>: a notation for representing Gregorian chant using ASCII characters</li>
+<li><a href="./gregoriotex/">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></a>: a <span class="tex">T<span class="epsilon">e</span>X</span> style for typesetting scores</li>
+<li>a software application to convert from gabc to Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></li>
 </ul>
 
-<p>Together, these tools, added to a <a href="/gregoriotex/tex.html">TeX</a> installation, allow the user to engrave beautiful Gregorian chant scores.</p>
+<p>Together, these tools, added to a <a href="./gregoriotex/tex.html"><span class="tex">T<span class="epsilon">e</span>X</span></a> installation, allow the user to engrave beautiful Gregorian chant scores.</p>
 
 <div class="legende">
-<img src="illus/Alleluia.png" title="" alt="" />
+<img src="./illus/Alleluia.png" title="" alt="" />
 </div>
 
 <p>The Gregorio project consists of 100% <a href="https://en.wikipedia.org/wiki/Free_software">free software</a>.</p>
 
-<p>This website contains an introduction, documentation and tutorial for Gregorio versions 3.0 and later. For information on older versions, please see the <a href="http://home.gna.org/gregorio/">old website</a>.</p>
+<p>This website contains an introduction, documentation and tutorial for Gregorio versions 4.0 and later.  For information on version 3.0, you will need to access the history in the <a href="https://github.com/gregorio-project/gregorio-project.github.io">GitHub repository</a> for this website.  For information on older versions (2.4.2 and earlier), please see the <a href="http://home.gna.org/gregorio/">old website</a>.</p>
```

Comment:
I pointed this link at the website repository specifically with the idea that it was a way for people to find older versions of the website, as the website version which is compatible with 3.0 won't be reachable the way the 2.3 website is.  Perhaps that's not clear though and some alternate wording is needed.